### PR TITLE
Pass list of ignored rules and hadolint flags

### PR DESCRIPTION
**Pull request by adnikiforov**
Added ability to:
* pass list of ignored rules (i.e. `DL3018 DL3009`)
* pass hadolint flags in general (i.e. `--trusted-registry docker.io`)

Updated readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,22 @@ With `reporter: github-pr-review` a comment is added to the Pull Request Convers
...
 
+### `hadolint_flags`
+
+Optional. Pass hadolint flags:
+```
+with:
+  hadolint_flags: --thrusted-repository docker.io
+```
+
+### `hadolint_ignore`
```
Comment (mgrachev) [inline README.md:42]: Please add these flags to the [action.yml](https://github.com/reviewdog/action-hadolint/blob/master/action.yml) file too.

**Review by mgrachev — COMMENTED** [review]
@adnikiforov 👋 Thank you for your contribution! 👍

I've left a comment. Please look at it 👀

File: README.md (hunk 1 above)
Comment (adnikiforov) [reply README.md:42]: Yep just did that, same thoughts.

File: action.yml [hunk 2]
```diff
@@ -5,6 +5,12 @@ inputs:
   github_token:
     description: 'GITHUB_TOKEN.'
     default: '${{ github.token }}'
+  hadolint_flags:
+    description: 'Hadolint flags. (hadolint <rubocop_flags>)'
```
Comment (mgrachev) [inline action.yml:9]:
```suggestion
    description: 'Hadolint flags. (hadolint <hadolint_flags>)'
```

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@adnikiforov Excellent work! 💪 Thank you! 🙏

**Pull request merged**